rundeckapp/grails-app/controllers/rundeck/controllers/ExecutionController.groovy

```diff
@@ -2290,15 +2290,19 @@ setTimeout(function(){
 
         def executionStatus = configurationService.executionModeActive
         int respStatus = executionStatus ? HttpServletResponse.SC_OK : HttpServletResponse.SC_SERVICE_UNAVAILABLE
+        boolean apiVersionAfterV35 = request.api_version > ApiVersions.V35
+        if(apiVersionAfterV35 && !executionStatus) {
+            respStatus =  params.passiveAs503 ? HttpServletResponse.SC_SERVICE_UNAVAILABLE : HttpServletResponse.SC_OK
```

Comment:
```suggestion
            respStatus =  params.boolean('passiveAs503',false) ? HttpServletResponse.SC_SERVICE_UNAVAILABLE : HttpServletResponse.SC_OK
```

problem: `params.passiveAs503` will resolve to true if any value is present (groovy truth)